apps/common/contexts/useWallet.tsx

```diff
@@ -45,14 +45,33 @@ export const WalletContextApp = memo(function WalletContextApp({children}: {chil
 			return [];
 		}
 		const	tokens: TUseBalancesTokens[] = [];
+		const	tokensExists: TDict<boolean> = {};
+
+		const	extraTokens = [
+			ETH_TOKEN_ADDRESS,
+			YCRV_TOKEN_ADDRESS,
+			LPYCRV_TOKEN_ADDRESS,
+			CRV_TOKEN_ADDRESS,
+			YVBOOST_TOKEN_ADDRESS,
+			YVECRV_TOKEN_ADDRESS,
+			CVXCRV_TOKEN_ADDRESS
+		];
+		for (const token of extraTokens) {
+			tokensExists[token] = true;
+			tokens.push({token});
+		}
+
 		Object.values(vaults || {}).forEach((vault?: TYearnVault): void => {
 			if (!vault) {
 				return;
 			}
-			tokens.push({token: vault?.address});
-			tokens.push({token: vault.token.address});
+			if (!tokensExists[toAddress(vault?.address)]) {
+				tokens.push({token: vault.address});
+			}
```

Comment:
if `vault?.address` is `undefined`, are we pushing to the `tokens` array an object `{ token: undefined }`?
```suggestion
			if (vault?.address && !tokensExists[toAddress(vault.address)]) {
				tokens.push({token: vault.address});
			}
```